help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md

```diff
@@ -0,0 +1,965 @@
+solution: Experience Platform
+title: Create a source connection and dataflow for SugarCRM Accounts & Contacts using the Flow Service API
+topic-legacy: tutorial
+description: Learn how to connect Adobe Experience Platform to SugarCRM Accounts & Contacts using the Flow Service API.
+hide: true
+hidefromtoc: true
+---
+# Create a source connection and dataflow for [!DNL SugarCRM Accounts & Contacts] using the Flow Service API
+
+[[!DNL SugarCRM]](https://www.sugarcrm.com/) is a customer relationship management (CRM) system. [!DNL SugarCRM]'s functionality includes sales-force automation, marketing campaigns, customer support, collaboration, Mobile CRM, Social CRM and reporting.
+
+To retrieve relationship data setup within [!DNL SugarCRM], this Adobe Experience Platform [source](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) connector leverages the below [!DNL SugarCRM] APIs :
+   1. [Account API](https://market.apidocs.sugarcrm.com/#b0aeb0cd-80ea-4688-8474-54e4873f32f3) to retrieve your companies, and the 
+   1. [Contact API](https://market.apidocs.sugarcrm.com/#308c5025-9478-4de3-8a41-1fc3cff1d8d1) to retrieve the people. 
+
+After you bring that data to Experience Platform, you can then execute any analytics.
+
+[!DNL SugarCRM Accounts & Contacts] uses bearer tokens as an authentication mechanism to communicate with the [!DNL SugarCRM] Account and Contact APIs.
+
+## Prerequisites
+
+The first step in creating a [!DNL SugarCRM Accounts & Contacts] source connection is to ensure that you have the below :
+1. A [!DNL SugarMarket] *(SugarCRM’s product for Marketing Automation)* account.
+1. Additionally, a unique API username and user account separate from any user account associated with the marketing or sales process; having API Access permissions in order to access the API. The process to obtain / set up the account is documented on the [[!DNL SugarMarket RESTFUL API (sugarcrm.com)]](https://market.apidocs.sugarcrm.com/#intro) page.
+
+### Gather required credentials
+
+In order to connect [!DNL SugarCRM Accounts & Contacts] to Platform, you must provide values for the following connection properties:
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| *`Host`* | API endpoint. | `developer.salesfusion.com` |
+| *`Username`* | Your SugarCRM developer account username. | `abc.def@example.com@sugarmarketdemo000.com` |
+| *`Password`* | Your SugarCRM developer account password. | `123456789` |
+
+## Connect [!DNL SugarCRM Accounts & Contacts] to Platform using the [!DNL Flow Service] API
+
+The following tutorial walks you through the steps to create a [!DNL SugarCRM Accounts & Contacts] source connection and create a dataflow to bring [!DNL SugarCRM Accounts & Contacts] data to Platform using the [[!DNL Flow Service] API](https://www.adobe.io/experience-platform-apis/references/flow-service/).
```

Comment:
we can shorten the middle of this to: "...to create a [!DNL SugarCRM Accounts & Contacts] source connection and dataflow to bring..."